### ISSUE TITLE
Positional vs named arguments on PHP 8

### ISSUE THREAD

**Issue by vincentchalamon** [Behat/Behat#1325]
Hi,

I'm making some tests with Behat on PHP 8 (rc).

The [MixedArgumentOrganiser](https://github.com/Behat/Behat/blob/master/src/Behat/Testwork/Argument/MixedArgumentOrganiser.php) will sometimes inject numered arguments mixed with named arguments, which throws the following error:
> Fatal error: Cannot use positional argument after named argument

I was wondering why use numered arguments instead of named arguments?

**Comment by ciaranmcnulty** [comment]
This should be fixed in #1313 - review is welcome

Position is simply the last fallback once type and name have failed to match, partly for legacy regex reasons:

``` 
/**
 * @Given /I have (\d+) bananas/
 */
function bananas ($number)
```

**Comment by vincentchalamon** [comment]
Closing as will be resolved in https://github.com/Behat/Behat/pull/1313
